### PR TITLE
Optional attendees (Step 4)

### PR DESCRIPTION
**Pull request by hadleyshapland**
Adding functionality for optional attendees and new tests to make sure it works.

### CODE REVIEW

File: walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java [hunk 1]
```diff
@@ -311,4 +313,167 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void optionalIsBusy() {
```
Comment (jerrychangusa) [inline walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java]: nit: maybe name this optionalAttendeeIsBusy()

**Review by jerrychangusa — APPROVED** [review]
LGTM

File: walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java (hunk 1 above)
Comment (hadleyshapland) [reply walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java]: good idea, fixed this

**Review by ghandre — APPROVED** [review]
Looks good, the explanations at the start of each test are helpful.

**Pull request merged**